superset-frontend/src/explore/components/controls/FilterControl/AdhocFilterEditPopoverSimpleTabContent/index.tsx

```diff
@@ -0,0 +1,476 @@
+      subject = option.column_name;
+      clause = CLAUSES.WHERE;
+    } else if (option && 'saved_metric_name' in option) {
+      subject = option.saved_metric_name;
+      clause = CLAUSES.HAVING;
+    } else if (option && option.label) {
+      subject = option.label;
+      clause = CLAUSES.HAVING;
+    }
+    const { operator, operatorId } = props.adhocFilter;
+    props.onChange(
+      props.adhocFilter.duplicateWith({
+        subject,
+        clause,
+        operator:
+          operator && isOperatorRelevant(operatorId, subject)
+            ? OPERATOR_MAPPING[operatorId].operation
+            : null,
+        expressionType: EXPRESSION_TYPES.SIMPLE,
+        operatorId,
+      }),
+    );
+  };
+  const onOperatorChange = (operatorId: Operators) => {
+    const currentComparator = props.adhocFilter.comparator;
+    let newComparator;
+    // convert between list of comparators and individual comparators
+    // (e.g. `in ('North America', 'Africa')` to `== 'North America'`)
+    if (MULTI_OPERATORS.has(operatorId)) {
+      newComparator = Array.isArray(currentComparator)
+        ? currentComparator
+        : [currentComparator].filter(element => element);
+    } else {
+      newComparator = Array.isArray(currentComparator)
+        ? currentComparator[0]
+        : currentComparator;
+    }
+    if (operatorId === Operators.IS_TRUE || operatorId === Operators.IS_FALSE) {
+      newComparator = Operators.IS_TRUE === operatorId;
+    }
+    if (operatorId && CUSTOM_OPERATORS.has(operatorId)) {
+      props.onChange(
+        props.adhocFilter.duplicateWith({
+          subject: props.adhocFilter.subject,
+          clause: CLAUSES.WHERE,
+          operatorId,
+          operator: OPERATOR_MAPPING[operatorId].operation,
+          expressionType: EXPRESSION_TYPES.SQL,
+          datasource: props.datasource,
+        }),
+      );
+    } else {
+      props.onChange(
+        props.adhocFilter.duplicateWith({
+          operatorId,
+          operator: OPERATOR_MAPPING[operatorId].operation,
+          comparator: newComparator,
+          expressionType: EXPRESSION_TYPES.SIMPLE,
+        }),
+      );
+    }
+  };
+  const onComparatorChange = (comparator: string) => {
+    props.onChange(
+      props.adhocFilter.duplicateWith({
+        comparator,
+        expressionType: EXPRESSION_TYPES.SIMPLE,
+      }),
+    );
+  };
+  return {
+    onSubjectChange,
+    onOperatorChange,
+    onComparatorChange,
+    isOperatorRelevant,
+  };
+};
+
+const AdhocFilterEditPopoverSimpleTabContent: React.FC<Props> = props => {
+  const selectProps = {
+    name: 'select-column',
+    showSearch: true,
+  };
+  const {
+    onSubjectChange,
+    onOperatorChange,
+    isOperatorRelevant,
+    onComparatorChange,
+  } = useSimpleTabFilterProps(props);
+  const [suggestions, setSuggestions] = useState<JsonObject>([]);
+  const [abortActiveRequest, setAbortActiveRequest] = useState<
+    (() => void) | null
+  >(null);
```

Comment:
maybe out of scope, but initializing this to `NO_OP` (or maybe `noOp`, I forget how it's named in our utils file, it's basically an empty function) would prevent the need to manually set the type and to check if `abortActiveRequest` exists before calling it everytime. Seems cleaner